Kitodo/src/test/java/org/kitodo/production/exporter/download/ExportMetsIT.java

```diff
@@ -100,7 +100,6 @@ public void exportMetsTest() throws Exception {
             fileService.createDirectory(ConfigCore.getUriParameter(ParameterCore.DIR_USERS), userDirectory);
         }
 
-        Assert.assertFalse("exportedFlag of process should be false", process.isExported());
```

Comment:
They don't work anymore because I changed it, and I think it should be like this: The exported flag should only be set when exported to DMS, but not if an export to the home directory is carried out. The flag should not be set when exporting to the home directory, isn’t it? That's why I removed the tests for it, too.